src/pool/srv_pool.c

```diff
@@ -580,11 +580,26 @@ ds_pool_svc_create(const uuid_t pool_uuid, int ntargets, const char *group,
 	struct dss_module_info *info = dss_get_module_info();
 	crt_endpoint_t		ep;
 	crt_rpc_t	       *rpc;
+	struct daos_prop_entry *lbl_ent;
+	struct daos_prop_entry *def_lbl_ent;
 	struct pool_create_in  *in;
 	struct pool_create_out *out;
 	struct d_backoff_seq	backoff_seq;
 	int			rc;
 
+	/* Check for default label property supplied */
```

Comment:
I wonder if this check might be better done in the calling function ds_mgmt_create_pool() prior to it invoking ds_mgmt_tgt_pool_create_ranks() (distributed allocation) so that if we do have invalid input that the corresponding distributed cleanup ds_mgmt_tgt_pool_destroy_ranks() will not be needed.